README.md

```diff
@@ -1,3 +1,5 @@
+> **Note**: We do not recommend using the charts in this repository directly except for development. Instead, please see https://artifacthub.io/packages/search?repo=spiffe
```

Comment:
Remove what to do here, so we can say "this is beta" and THEN "don't directly link"

```suggestion
```